geoopt/manifolds/lorentz/__init__.py

```diff
@@ -159,6 +157,7 @@ def transp_follow_expmap(
     def geodesic_unit(
         self, t: torch.Tensor, x: torch.Tensor, u: torch.Tensor, *, dim=-1, project=True
     ) -> torch.Tensor:
+        # FIXME: project U
```

Comment:
@rrkarim is this intended? I missed this in my PR

Reply:
you want to add additional argument to project on the tangent space?

Reply:
You can see in the example notebook, I was projecting u before parallel transport.

Reply:
I was confused about this. I think the argument should already belong to tangent space, protection should be on user side.